### PR TITLE
add user permissions splainer

### PR DESCRIPTION
**Pull request by jrday**
document the slack and github scopes requested by Atomist

### CODE REVIEW

File: docs/user-guide/permissions/github.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Atomist automates activities like project creation and code changes that operate on code in GitHub repos. Atomist also makes it easy to get relevant, actionable notification of GitHub activity like new commits or issues. In order to do that, the Atomist GitHub integration requests the following permissions:
```
Comment (whostolebenfrog) [inline docs/user-guide/permissions/github.md]: "In order to do that, the Atomist GitHub integration requests the following permissions:"

I'd replace this with:

"To enable this, the Atomist GitHub integration requests the following permissions:"

File: docs/user-guide/permissions/github.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+GitHub Scope |	How Atomist Uses It
+------|--------
+`repo` | Enables Atomist to read and write code in your repos, and get activity notifications for commits.
+`read:org` | Read information about the GitHub org, like teams and members of the org.
+`admin:repo_hook` | In order to receive activity alerts, Atomist can configure a webhook on the repos of your choice.
```
Comment (whostolebenfrog) [inline docs/user-guide/permissions/github.md]: @slimslenderslacks Has been talking about trying to downplay using repo hooks except where required on personal orgs.

With that in mind we could rephrase this:

"Allows atomist to configure a webhook on a single repo and start receiving activity alerts"

File: docs/user-guide/permissions/slack.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* Listen for messages and commands directed at the bot
+* Send notifications in channels that the bot has been invited to
+* Send notifications via direct message
+
+When you authorize the Atomist Slack app, it requests the following permissions. For the bot to operate properly, all of these scopes need to be granted.
+
+Slack Scope | How Atomist Uses It
+-------------|----------
+`bot` | Enables the bot to hear and respond to channel activity, such as certain bot commands and events.
+`identify` | Used to confirm your identity so that the bot knows who to address.
```
Comment (whostolebenfrog) [inline docs/user-guide/permissions/slack.md]: I would say "Used to confirm your identity and test the authentication"

File: docs/user-guide/permissions/slack.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+`channels:read` |  Allows the bot to list channels in the team.
+`channels:write` | Enables the bot to create channels and write messages to channels. For example, when you create a new project with Atomist, the bot can create a channel that will be associated with the repo.
+`im:read` | Allows the bot to list direct message channels associated with the bot users. Sometimes Atomist communicates with users via direct message.
+`team:read` | Lets Atomist know basic information about the Slack team, like its ID, name, domain.
```
Comment (whostolebenfrog) [inline docs/user-guide/permissions/slack.md]: and between name and domain?

File: docs/user-guide/permissions/github.md (hunk 1 above)
Comment (ddgenome) [inline docs/user-guide/permissions/github.md]: As an English teacher once told me, "In order" is always superfluous.

File: docs/user-guide/permissions/index.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Atomist integrates with and automates activities over various tools and services in your development toolchain. To do so, certain permission are needed. We aim to ask only for the permissions required to get the job done.
+
+Please see the individual sections for specifics on the permissions requested by Atomist.
+
+* [Slack App OAuth Scopes](slack.md)
+* [GitHub OAuth Scopes](github.md)
+
+If you still have questions about permissions, please contact us in our [Atomist Community slack team](https://join.atomist.com) or email us at support@atomist.com.
```
Comment (ddgenome) [inline docs/user-guide/permissions/index.md]: Would be good for the email address to be a mailto: link.

File: docs/user-guide/permissions/index.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Atomist integrates with and automates activities over various tools and services in your development toolchain. To do so, certain permission are needed. We aim to ask only for the permissions required to get the job done.
+
+Please see the individual sections for specifics on the permissions requested by Atomist.
+
+* [Slack App OAuth Scopes](slack.md)
```
Comment (ddgenome) [inline docs/user-guide/permissions/index.md]: Remove "App"?

File: docs/user-guide/permissions/index.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+* [GitHub OAuth Scopes](github.md)
```
Comment (ddgenome) [inline docs/user-guide/permissions/index.md]: Technically the text for list items is supposed to start in column 4.  MkDocs and GitHub are forgiving on this, some Markdown renderers are not. Keeping with the standard allows more easy migration away from MkDocs should we ever need to.

File: docs/user-guide/permissions/slack.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+Atomist acts as a bot in your Slack team, and can take these actions:
+
+* Listen for messages and commands directed at the bot
+* Send notifications in channels that the bot has been invited to
+* Send notifications via direct message
+
+When you authorize the Atomist Slack app, it requests the following permissions. For the bot to operate properly, all of these scopes need to be granted.
```
Comment (ddgenome) [inline docs/user-guide/permissions/slack.md]: `s/need to/must/`

File: docs/user-guide/permissions/slack.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+Slack Scope | How Atomist Uses It
+-------------|----------
+`bot` | Enables the bot to hear and respond to channel activity, such as certain bot commands and events.
```
Comment (ddgenome) [inline docs/user-guide/permissions/slack.md]: `s/certain //`

File: docs/user-guide/permissions/slack.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+`identify` | Used to confirm your identity so that the bot knows who to address.
+`channels:read` |  Allows the bot to list channels in the team.
+`channels:write` | Enables the bot to create channels and write messages to channels. For example, when you create a new project with Atomist, the bot can create a channel that will be associated with the repo.
```
Comment (ddgenome) [inline docs/user-guide/permissions/slack.md]: "... will be associated with the project's repository."

File: docs/user-guide/permissions/slack.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+`im:read` | Allows the bot to list direct message channels associated with the bot users. Sometimes Atomist communicates with users via direct message.
```
Comment (ddgenome) [inline docs/user-guide/permissions/slack.md]: "associated with the bot users"? Should that just be "associated with users" or perhaps just removed altogether?

File: docs/user-guide/permissions/slack.md (hunk 8 above)
Comment (ddgenome) [inline docs/user-guide/permissions/slack.md]: Perhaps move the second sentence to after the table?

File: docs/user-guide/permissions/index.md [hunk 12]
```diff
@@ -0,0 +1,8 @@
+Atomist integrates with and automates activities over various tools and services in your development toolchain. To do so, certain permission are needed. We aim to ask only for the permissions required to get the job done.
```
Comment (ddgenome) [inline docs/user-guide/permissions/index.md]: Suggestion for the last sentence: "Atomist only asks for permissions required to accomplish its work."

File: docs/user-guide/permissions/github.md [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+`admin:org_hook` | To receive activity alerts for all repos in an org, Atomist can configure an org-level webhook.
```
Comment (ddgenome) [inline docs/user-guide/permissions/github.md]: `s/repos/repositories/`

File: docs/user-guide/permissions/github.md (hunk 1 above)
Comment (ddgenome) [inline docs/user-guide/permissions/github.md]: `s/repos/repositories/`

File: docs/user-guide/permissions/github.md [hunk 14]
```diff
@@ -0,0 +1,12 @@
+Atomist automates activities like project creation and code changes that operate on code in GitHub repos. Atomist also makes it easy to get relevant, actionable notification of GitHub activity like new commits or issues. In order to do that, the Atomist GitHub integration requests the following permissions:
+
+GitHub Scope |	How Atomist Uses It
+------|--------
+`repo` | Enables Atomist to read and write code in your repos, and get activity notifications for commits.
```
Comment (ddgenome) [inline docs/user-guide/permissions/github.md]: `s/repos/repositories/`

File: docs/user-guide/permissions/github.md (hunk 2 above)
Comment (ddgenome) [inline docs/user-guide/permissions/github.md]: `s/repos/repositories/`

File: docs/user-guide/permissions/github.md (hunk 1 above)
Comment (ddgenome) [inline docs/user-guide/permissions/github.md]: `s/notification/notifications/`

**Pull request merged**